tests/integration/test_enrollment/data/wazuh_enrollment_tests.yaml

```diff
@@ -0,0 +1,273 @@
+-
+  name: "Default Name"
+  description: "No agent name in auto enrollment configuration"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}'\n"
+    response: "OSSEC K:'001 {host_name} any TopSecret'\n"
+-
+  name: "Valid agent address"
+  description: "Check a valid agent_address configurations"
```

Comment:
Configuration instead of configurations?

Reply:
Fixed